apps/financial-aid/backend/src/app/modules/application/application.controller.ts

```diff
@@ -121,4 +133,29 @@ export class ApplicationController {
   ): Promise<ApplicationModel> {
     return this.applicationService.create(application, user)
   }
+
+  @Post('applicationEvent')
+  @ApiCreatedResponse({
+    type: ApplicationEventModel,
+    description: 'Creates a new application event',
+  })
+  async createEvent(
+    @Body() applicationEvent: CreateApplicationEventDto,
+  ): Promise<ApplicationModel> {
+    const createApplicationEvent = await this.applicationEventService.create(
```

Comment:
We don't use this variable, should we just throw it into the bin 🗑️  and do `await this.applicationEventService.create(...)` 